Firestore/Example/Tests/Local/FSTLocalSerializerTests.mm

```diff
@@ -56,7 +56,9 @@ - (GCFSValue *)encodedInteger:(int64_t)value;
 - (GCFSValue *)encodedString:(NSString *)value;
 @end
 
-@interface FSTLocalSerializerTests : XCTestCase
+@interface FSTLocalSerializerTests : XCTestCase {
+  DatabaseId databaseId;
```

Comment:
This should be `_databaseID`. Here and throughout.

Reply:
done